release-notes/README.md

```diff
@@ -4,6 +4,16 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## May 9, 2022
+
+### iOS AEPCore 3.6.0
+
+* App Extension Support
```

Comment:
What does this mean? It's a bit vague for my liking.

Reply:
Support for using the SDK within App Extensions. https://developer.apple.com/app-extensions/

Reply:
I'd use the full sentence instead then: "Support using the SDK within App Extensions" instead, since that's a bit less vague.

Reply:
Is what i added ok?

Reply:
Yup! Sorry, I added this when it was still the outdated version.